src/Frontend/Modules/Profiles/Widgets/SecurePage.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Frontend\Modules\Profiles\Widgets;
+
+/*
+ * This file is part of Fork CMS.
+ *
+ * For the full copyright and license information, please view the license
+ * file that was distributed with this source code.
+ */
+
+use Common\Exception\RedirectException;
+use Frontend\Core\Engine\Base\Widget as FrontendBaseWidget;
+use Frontend\Core\Engine\Navigation;
+use Frontend\Modules\Profiles\Engine\Authentication as FrontendProfilesAuthentication;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+
+/**
+ * This is a widget to help you secure a page and make it only accessible for logged-in users.
+ */
+class SecurePage extends FrontendBaseWidget
+{
+    /**
+     * Execute the extra
+     */
+    public function execute()
+    {
+        parent::execute();
+        $this->loadTemplate();
```

Comment:
I'm afraid not @WouterSioen . When I removed this line, it complained. And there doesn't exist a parameter to pass, to indicate there's no template needed. NULL is the default param and then it uses the template with the same name as the action/widget.